### PR TITLE
Refactor volvooncall to use ConfigFlow

### PR DESCRIPTION
**Pull request by y34hbuddy**
## Proposed change

Remove all documentation for YAML configuration and replace with ConfigFlow.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [X] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [X] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/76680/

## Checklist

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [X] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/volvooncall.markdown [hunk 1]
```diff
@@ -20,105 +20,4 @@ ha_integration_type: integration
...
-scandinavian_miles:
-  description: If set to true, Scandinavian miles ("mil") are used for distances and fuel range.
-  required: false
-  type: boolean
-  default: false
-{% endconfiguration %}
-
-### Available Resources
-
-The list of currently available resources:
```
Comment (frenck) [inline source/_integrations/volvooncall.markdown]: Why are these removed in the process?

**Review by frenck — DISMISSED** [review]

File: source/_integrations/volvooncall.markdown (hunk 1 above)
Comment (y34hbuddy) [reply source/_integrations/volvooncall.markdown]: I have not yet provided a way to custom-select resources via ConfigFlow. I intend on providing that functionality in a future PR.
If you prefer to keep the list in the documentation (despite not being able to configure it), I'm happy to keep the list.

Comment (MartinHjelmare) [reply source/_integrations/volvooncall.markdown]: We don't need a config flow for that. The user can disable entities that they don't need.

Comment (y34hbuddy) [reply source/_integrations/volvooncall.markdown]: Ah indeed, thanks for the reminder Martin. So what is the recommendation here? Should we keep this list in the docs or remove it?

Comment (frenck) [reply source/_integrations/volvooncall.markdown]: I think, in general, it makes sense to tell a user what they can expect from an integration to provide/make available in the documentation.

Comment (y34hbuddy) [reply source/_integrations/volvooncall.markdown]: Good feedback, thanks @frenck!

**Review by klaasnicolaas — APPROVED** [review]
✅ LGTM
